data/sql/updates/pending_db_world/rev_1598881374545560300.sql

```diff
@@ -0,0 +1,15 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598881374545560300');
+/*
+ * General: NPC Update
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* Content 3.3.5 */
+
+-- NPC entry 40405 Kieupid, Pet Trainer in Silvermoon City (map: 530, zone ID 0, area ID 0)
+SET @CGUID = 248600;
+
+DELETE FROM `creature` WHERE `id` = 40405;
```

Comment:
```suggestion
DELETE FROM `creature` WHERE `id` = 40405 AND `guid` = @CGUID;
```

You ALWAYS need to delete the Primary Key.